pkg/k3d/registry.go

```diff
@@ -249,3 +261,74 @@ func CacheImageInTheBackground(ctx context.Context, image string) error {
 	log.SendCloudReport("info", "Successfully cached images in background", "Running", nil)
 	return dockerClient.ContainerExecStart(ctx, exec.ID, dockerTypes.ExecStartCheck{})
 }
+
+func CheckDockerRequirements() error {
+	if os.Getenv("DISABLE_DOCKER_CHECKS") == "true" {
+		return nil
+	}
+	_, err := exec.LookPath("docker")
+	if err != nil {
+		return errors.New("Docker is not installed. Docker is required, please install it and retry. https://docs.docker.com/engine/install/")
```

Comment:
consider: Docker is required => Docker is prerequisite

Reply:
Updated